### PR TITLE
Redesign Blocks Selector & Other Improvements

### PR DESCRIPTION
**Pull request by trindadedev13**
also move my package to pro sketchware

### CODE REVIEW

File: app/src/main/java/a/a/a/aB.java [hunk 1]
```diff
@@ -23,18 +23,19 @@
...
-    private boolean msgIsSelectable = false;
-    private View dialogCustomView;
-    private View dialogButtonsContainer;
-    private String dialogDefaultText = "Default";
-    private View.OnClickListener dialogDefaultListener = null;
-    private String dialogNoText = "No";
-    private View.OnClickListener dialogNoListener = null;
-    private String dialogYesText = "Yes";
-    private View.OnClickListener dialogYesListener = null;
+    // I don't know if it's bad  make everything public, but I did this so I could use .aB.apply in kotlin
```
Comment (hasrat-ali) [inline app/src/main/java/a/a/a/aB.java:26]: Fuck fuck kotlin

Comment (trindadedev13) [reply app/src/main/java/a/a/a/aB.java:26]: what

Comment (SyntaxSpin) [reply app/src/main/java/a/a/a/aB.java:26]: They love java keep it java

Comment (trindadedev13) [reply app/src/main/java/a/a/a/aB.java:26]: @SyntaxSpin First tell me where did I say that I will recreate aB in kotlin?

Comment (SyntaxSpin) [reply app/src/main/java/a/a/a/aB.java:26]: Hasrat Said it idk

Comment (trindadedev13) [reply app/src/main/java/a/a/a/aB.java:26]: nah, he said "fuck fuck kotlin"

File: app/src/main/java/a/a/a/aB.java
Comment (trindadedev13) [inline app/src/main/java/a/a/a/aB.java:1]: For anyone reviewing, please let this change go through, I don't want to have to redo the dialogs using the standard methods (dialog.a(...) etc...)🙏🙏🙏🥺🥺

Comment (trindadedev13) [reply app/src/main/java/a/a/a/aB.java:1]: researching about it, I realized that there is no difference then gg

**Comment by trindadedev13** [comment]
ready for review, I did the tests and it is working correctly.

**Pull request merged**